src/main/java/com/monumental/util/application/InitialDataLoadApplication.java

```diff
@@ -19,16 +27,23 @@
  * Then does validation on each CsvMonumentConverterResult's Monument before inserting it into the database
  * Finally, closes the CsvFileReader
  */
+@Configuration
+@SpringBootApplication
+@ComponentScan("com.monumental")
 public class InitialDataLoadApplication {
+
     public static void main(String[] args) {
-        String pathToDatasetCsv = "C:\\Users\\nickb\\Documents\\Initial Dataset.csv";
+        SpringApplication app = new SpringApplication(InitialDataLoadApplication.class);
+        // Don't start a web server - allows this application to run while the main Application is running
+        app.setWebApplicationType(WebApplicationType.NONE);
```

Comment:
And this prevents the actual web server from starting on port 8080 so you don't get an error if the server is also running (important for the VM to be able to run it)